pkgs/development/tools/misc/bashdb/default.nix

```diff
@@ -1,14 +1,30 @@
-{ lib, stdenv, fetchurl, makeWrapper, python3Packages }:
+{ lib
+, stdenv
+, fetchurl
+, fetchpatch
+, makeWrapper
+, python3Packages
+}:
 
 stdenv.mkDerivation rec {
   pname = "bashdb";
-  version = "4.4-1.0.0";
+  version = "5.0-1.1.2";
 
   src = fetchurl {
     url =  "mirror://sourceforge/bashdb/${pname}-${version}.tar.bz2";
-    sha256 = "0p7i7bpzs6q1i7swnkr89kxqgzr146xw8d2acmqwqbslzm9dqlml";
+    sha256 = "sha256-MBdtKtKMWwCy4tIcXqGu+PuvQKj52fcjxnxgUx87czA=";
   };
 
+  patches = [
+    # Enable building with bash 5.1/5.2
```

Comment:
just a nit, feel free to ignore. it seems that you are talking about bash 5.2 . Which doesn't exist I think.

Reply:
Yeah the patch adds an explicit allow for bash 5.2. It's a fix for a potential future bash 5.2 upgrade

Reply:
hmm, well if upstream says so, then nothing we should do about it.